apps/oc_chef_wm/src/chef_wm_malformed.erl

```diff
@@ -47,6 +48,8 @@ malformed_request_message(bad_clock, Req, State) ->
     Msg = iolist_to_binary([<<"Failed to authenticate as ">>, User,
                             <<". Synchronize the clock on your host.">>]),
     {[{<<"error">>, [Msg]}]};
+malformed_request_message(bad_date, _Req, _State) ->
```

Comment:
Probably a bit nicer, I had it general because I didn't want to pass the field to chef_regex, but when I wrote it that way, I didn't realize we don't really use the chef_regex errors, so I'll just move the bad date error message macro somewhere else.